lua/kznllm/init.lua

```diff
@@ -101,11 +101,6 @@ function M.get_visual_selection(opts)
   -- in visual block and visual line mode, we expect first column of srow and last column of erow
   if mode == 'V' or mode == '\22' or mode == 'n' then
     scol, ecol = 0, -1
-  else
-    -- in visual mode we need to include the last column of erow
-    -- capped at the row length, as visual mode can move the cursor past the row end
-    local erow_content = vim.api.nvim_buf_get_lines(0, erow, erow + 1, false)[1]
-    ecol = math.min(ecol + 1, #erow_content)
```

Comment:
i think we can get rid of this completely, the erow + 1 adjustment doesn't seem to be right in the first place now that I'm poking around with it